include/exadg/acoustic_conservation_equations/throughput.h

```diff
@@ -0,0 +1,201 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef EXADG_ACOUSTIC_CONSERVATION_EQUATIONS_THROUGHPUT_H_
+#define EXADG_ACOUSTIC_CONSERVATION_EQUATIONS_THROUGHPUT_H_
+
+// likwid
+#ifdef EXADG_WITH_LIKWID
+#  include <likwid.h>
+#endif
+
+// ExaDG
+
+// driver
+#include <exadg/acoustic_conservation_equations/driver.h>
+
+// utilities
+#include <exadg/operators/hypercube_resolution_parameters.h>
+#include <exadg/operators/throughput_parameters.h>
+#include <exadg/utilities/general_parameters.h>
+
+// application
+#include <exadg/acoustic_conservation_equations/user_interface/declare_get_application.h>
+
+namespace ExaDG
+{
+void
+create_input_file(std::string const & input_file)
+{
+  dealii::ParameterHandler prm;
+
+  GeneralParameters general;
+  general.add_parameters(prm);
+
+  HypercubeResolutionParameters resolution;
+  resolution.add_parameters(prm);
+
+  ThroughputParameters<Acoustics::OperatorType> throughput;
+  throughput.add_parameters(prm);
+
+  try
```

Comment:
Do you remember the reason for this try-catch block or why we do not have the same in the solver file?

I have seen that we also have this block in other modules like `ConvDiff`.

Reply:
I didn't understand why this try catch is used. I simply adapted to the rest of the modules. I am happy to remove it if you also do not see a need for it.

Reply:
Let's consider this a sub-ordinate problem for now, given discussion https://github.com/exadg/exadg/pull/592#discussion_r1389154815.